datanode/src/storage_interface.py

```diff
@@ -80,34 +108,30 @@ def __init__(self, connectionstring=DEFAULT_CONNECTION, testgroupid=None):
 
     Args:
       connectionstring:
-        Zookeeper connection string - server:port,server:port,...
+        cokcroach connection string - following postgres URL syntax
       testgroupid:
         ID to use if in test mode, none for normal mode
     """
     if testgroupid:
       self.set_testmode(testgroupid)
     if not connectionstring:
       connectionstring = DEFAULT_CONNECTION
-    log.debug('Creating metadata manager object and connecting to zookeeper...')
+    log.debug('Creating metadata manager object and connecting to cockroach...')
     try:
-      if set(BAD_CHARACTER_CHECK) & set(connectionstring):
-        raise ValueError
-      self.zk = KazooClient(hosts=connectionstring, timeout=CONNECTION_TIMEOUT)
-      self.zk.add_listener(self.zookeeper_connection_listener)
-      self.zk.start()
+      self.conn = psycopg2.connect(connectionstring)
+      with self.conn.cursor() as curs:
+        curs.execute(SQL_CREATE_TABLE)
```

Comment:
should we pull table creation into a separate file?

We can skip this and clean this up later for the reference implementation

Reply:
Absolutely, I think I noted that down in the PR description. We should handle schema migrations properly for the reference implementation. For the scope and purpose of this PR, I'm inclined to leave it in the `__init__` unless someone has strong objections.